Berksfile

```diff
@@ -2,39 +2,45 @@ source 'https://supermarket.chef.io'
 
 # OSL Base deps
 cookbook 'aliases', git: 'git@github.com:osuosl-cookbooks/aliases'
-cookbook 'apt', '>= 2.3.8'
-cookbook 'apache2', '< 2.0.0'
 cookbook 'base', git: 'git@github.com:osuosl-cookbooks/base'
-cookbook 'database', '>= 2.0.0'
 cookbook 'firewall', git: 'git@github.com:osuosl-cookbooks/firewall'
 cookbook 'modules', git: 'git@github.com:osuosl-cookbooks/modules-cookbook'
-cookbook 'monitoring', git: 'git@github.com:osuosl-cookbooks/monitoring'
-cookbook 'munin'
-cookbook 'omnibus_updater'
-cookbook 'osl-apache',
-         git: 'git@github.com:osuosl-cookbooks/osl-apache',
-         tag: 'v1.0.17'
-cookbook 'osl-munin', git: 'git@github.com:osuosl-cookbooks/osl-munin'
 cookbook 'osl-nrpe', git: 'git@github.com:osuosl-cookbooks/osl-nrpe'
-cookbook 'scl'
+cookbook 'osl-apache', git: 'git@github.com:osuosl-cookbooks/osl-apache'
+cookbook 'osl-munin', git: 'git@github.com:osuosl-cookbooks/osl-munin'
 cookbook 'resource_from_hash',
          git: 'git@github.com:osuosl-cookbooks/resource_from_hash'
-cookbook 'runit', '1.5.10'
 cookbook 'statsd', github: 'att-cloud/cookbook-statsd'
-cookbook 'yum', '>= 3.1.4'
-cookbook 'yum-epel', '>= 0.3.4'
-cookbook 'yum-fedora'
+cookbook 'yum-qemu-ev', git: 'git@github.com:osuosl-cookbooks/yum-qemu-ev.git'
+
+# WIP patches
+%w(
+  dashboard
+).each do |cb|
```

Comment:
It's nice to have if I have more than one cookbook I need to use in WIP. This should go away by the time we need to merge this cookbook (I hope).